kotlinpoet/src/main/java/com/squareup/kotlinpoet/tags/TypeAliasTag.kt

```diff
@@ -0,0 +1,10 @@
+package com.squareup.kotlinpoet.tags
+
+import com.squareup.kotlinpoet.TypeName
+
+/**
+ * This tag indicates that this [TypeName] represents a `typealias` type.
+ *
+ * @property [abbreviatedType] the underlying type for this alias.
+ */
+public class TypeAliasTag(public val abbreviatedType: TypeName)
```

Comment:
Would it make sense to make this a `value` class?

Reply:
Seeing as we have to let it by the class, I don't really know if there would be any value to it

Reply:
Ah, true